Fixing travis build

jainsat opened the pull request.

jainsat committed:
Patch to fix travis build

jainsat committed:
Commit to disable aborting travis build on found lint errors.

codelibra commented:
Satya,
Can you also comment, how will this fix the travis build?
Excluding lint errors and changing android version.

jainsat commented:
I haven't changed android version. I just added SDK version used to compile the project which was missing. It was shown in build error as " failed to find target android-22". So I added that and it worked.
